### ISSUE TITLE
CairoSVG 2.x.x distributed via pip for python 2.x

### ISSUE THREAD

**Issue by dougcunn** [Kozea/CairoSVG#145]
```
% pip search CairoSVG
CairoSVG (2.0.2)  - A Simple SVG Converter based on Cairo
  INSTALLED: 1.0.22
  LATEST:    2.0.2
```

**Comment by liZe** [comment]
I don't know if there's a solution about that…

**Comment by liZe** [comment]
It should be fixed in 2.0.3, if you use at least pip >= 9.0.0. I can't test right now as PyPI has some cache, I'll check that later.

Thanks for the report!

**Comment by dougcunn** [comment]
Thanks. No way to get the latest 1.x distributed for Python 2.x with earlier versions of pip?  Otherwise lot of new users to CairoSVG on Python 2.x will be lost.  Would be a shame. Such a useful module and still tons of people on Python 2.x.

**Comment by liZe** [comment]
> No way to get the latest 1.x distributed for Python 2.x with earlier versions of pip?

I don't think so. There's a [question on StackOverflow](http://stackoverflow.com/questions/42238484/prevent-package-from-being-installed-on-old-python-versions) about that.

Pip 9.0 is available on Python 2.6+ and CairoSVG is packaged for many distributions, let's hope that it's enough.

**Comment by dougcunn** [comment]
Ah.  Hopefully that's enough.
